inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -2,17 +2,23 @@
 import math
 import os
 import random
+import shutil
 
+from absl import logging
 from functools import singledispatchmethod
 from typing import Optional
 from uuid import UUID
 
 import numpy as np
 
-from inductiva import scenarios, simulation, stellarators, tasks
+from inductiva import scenarios, simulation, stellarators, tasks, types, utils
```

Comment:
try to do just:

```
import inductiva
```

All the code below becomes much more readable.

Reply:
We are having problems with the structure of the imports. Some are working to do just that other don't...

Reply:
I had that at first and I really wanted to use it, but in this case I think its impossible because it creates a circular import. This is due to the fact that in the inductiva `__init__.py` file I need to do `from . import stellarators`. Therefore, if here I do `import inductiva` it will create a circular import and the code will not run.

Reply:
I do not know if there is another way to make the code more readable and avoid the circular imports.